Lib/glyphsLib/__init__.py

```diff
@@ -39,6 +39,7 @@
     "build_masters", "build_instances", "load_to_ufos",
     "load", "loads", "dump", "dumps",
 )]
+__all__.extend(glyphsLib.classes.__all__)
```

Comment:
I get `NameError: name 'glyphsLib' is not defined`

Reply:
I didn't even test it :/ sorry, will try again!

Reply:
on py27, you also need to encode the `unicode_literals` with `tostr`.
I suggest you do

```python
from glyphsLib.classes import __all__ as __all_classes__

__all__ = [tostr(s) for s in [
    ...
] + __all_classes__]
```

Reply:
OK, should I do it also in classes.py? It has `unicode_literals` but doesn't take any precautions with `__all__`

Reply:
otherwise if you attempt to do `from glyphsLib import *` you get
```
TypeError: Item in ``from list'' must be str, not unicode
```

yeah, another weird python 2.7 bug.. https://bugs.python.org/issue21720

Reply:
>  should I do it also in classes.py? It has unicode_literals but doesn't take any precautions with __all__

that's the weird thing.. Even if the classes submodule imports unicode_literals, doing `from glyphsLib.classes import *` works without that TypeError. Only when it's in the top-level `__init__.py` it does that :-/